### PR TITLE
Revert "Error annotation"

### PR DESCRIPTION
**Pull request by Bibby0110**
Reverts yogstation13/Yogstation#8909

Doesn't actually, you know, work

### CODE REVIEW

File: .github/workflows/turdis.yml [hunk 1]
```diff
@@ -30,6 +30,9 @@ jobs:
       
       - name: Setup Rust
         uses: hecrj/setup-rust-action@v1.3.2
+        with:
```
Comment (alexkar598) [inline .github/workflows/turdis.yml]: no

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: .github/workflows/turdis.yml (hunk 1 above)
Comment (alexkar598) [reply .github/workflows/turdis.yml]: it doesnt take arguments anymore or something

**Comment by JamieD1** [comment]
Works fine:
image

**Comment by Bibby0110** [comment]
Doesn't work fine

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**